fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java

```diff
@@ -573,6 +573,11 @@ public void queryRetry(TUniqueId queryId) throws Exception {
         UUID uuid;
         int retryTime = Config.max_query_retry_time;
         retryTime = retryTime <= 0 ? 1 : retryTime + 1;
+        // If the query is an `outfile` statement,
+        // we execute it only once to avoid exporting redundant data.
+        if (parsedStmt instanceof Queriable) {
```

Comment:
better add a session var?

Reply:
We can not handle the retry of outfile, so I think it is not necessary to add variable to control it.